core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -567,7 +567,6 @@ abstract class AbstractControllerBrokerRequestBatch(config: KafkaConfig,
       }
     }
     leaderAndIsrRequestMap.clear()
-    updateType = LeaderAndIsrRequest.Type.UNKNOWN
```

Comment:
hmm, why do we need to remove this part?

Reply:
It just moved up one call (to sendRequestsToBrokers)

Reply:
ok